### PR TITLE
Update to WildFly 9.0.0.CR1 again

### PR DESCRIPTION
**Pull request by seanf**
- Add jboss-web.xml to Arquillian deployments to ensure JNDI resources
  are initialised.
- Rename zanataTestDatasource to zanataDatasource to match jboss-web.xml.
- Remove ds.jndi.name property.

### CODE REVIEW

File: zanata-war/src/main/webapp-jboss/WEB-INF/classes/components.properties [hunk 1]
```diff
@@ -3,5 +3,5 @@
 
 env.debug=${env.debug}
 env.dbunit.type=${env.dbunit.type}
-persistence-unit-name=${ds.jndi.name}PU
+persistence-unit-name=zanataDatasourcePU
```
Comment (carlosmunoz) [inline zanata-war/src/main/webapp-jboss/WEB-INF/classes/components.properties]: Do we still need this? Or can we just hardcode components.xml as well.

File: zanata-war/src/test/resources/arquillian/components.properties [hunk 2]
```diff
@@ -3,5 +3,5 @@
 
 env.debug=true
 env.dbunit.type=HSQL
-persistence-unit-name=zanataTestDatasourcePU
+persistence-unit-name=zanataDatasourcePU
```
Comment (carlosmunoz) [inline zanata-war/src/test/resources/arquillian/components.properties]: Same question here.

File: zanata-war/src/main/webapp-jboss/WEB-INF/classes/components.properties (hunk 1 above)
Comment (seanf) [inline zanata-war/src/main/webapp-jboss/WEB-INF/classes/components.properties]: Done.

**Comment by seanf** [comment]
@carlosmunoz I amended the original commit to additionally rename zanataTestDatasourcePU to zanataDatasourcePU.

**Comment by carlosmunoz** [comment]
:+1: Reviewed

**Comment by seanf** [comment]
Squash-merged as b133621822e3e9678f8f606f0429a24762133e66

**Pull request closed without merging**